question_panel/question_panel.gd

```diff
@@ -13,7 +13,7 @@ signal answer_selected(answer: bool)
 
 func display_question(difficulty):
 	var question_handler = preload("res://data_providers/resources/question_provider.tres")
-	
+	print ("Selected difficulty in display_question():" + str(difficulty))
```

Comment:
Don't leave your debug print statements in when it is not for demonstration purposes.

Reply:
Sorry about that